autoreject/tests/test_autoreject.py

```diff
@@ -127,62 +131,68 @@ def test_autoreject():
                           random_state=42)
     ar = LocalAutoRejectCV(cv=3, picks=picks, thresh_func=thresh_func,
                            n_interpolates=[1, 2],
-                           consensus_percs=[0.5, 1])
+                           consensus=[0.5, 1])
     assert_raises(AttributeError, ar.fit, X)
     assert_raises(ValueError, ar.transform, X)
     assert_raises(ValueError, ar.transform, epochs)
 
     ar.fit(epochs_fit)
-    fix_log = ar.fix_log
-    bad_epochs_idx = ar.local_reject_.bad_epochs_idx_
-    good_epochs_idx = ar.local_reject_.good_epochs_idx_
+    reject_log = ar.get_reject_log(epochs_fit)
     for ch_type in ch_types:
         # test that kappa & rho are selected
         assert_true(
             ar.n_interpolate_[ch_type] in ar.n_interpolates)
         assert_true(
-            ar.consensus_perc_[ch_type] in ar.consensus_percs)
-        # test that local autoreject is synced with AR-CV instance
-        assert_equal(
-            ar.n_interpolate_[ch_type],
-            ar.local_reject_.n_interpolate[ch_type])
-        assert_equal(
-            ar.consensus_perc_[ch_type],
-            ar.local_reject_.consensus_perc[ch_type])
+            ar.consensus_[ch_type] in ar.consensus)
+
+        assert_true(
+            ar.n_interpolate_[ch_type] ==
+            ar.local_reject_[ch_type].n_interpolate_[ch_type])
+        assert_true(
+            ar.consensus_[ch_type] ==
+            ar.local_reject_[ch_type].consensus_[ch_type])
 
     # test complementarity of goods and bads
-    assert_array_equal(
-        np.sort(np.r_[bad_epochs_idx, good_epochs_idx]),
-        np.arange(len(epochs_fit)))
+    assert_array_equal(len(reject_log.bad_epochs), len(epochs_fit))
 
     # test that transform does not change state of ar
-    epochs_fit.fit_ = True
     epochs_clean = ar.transform(epochs_fit)  # apply same data
```

Comment:
@dengemann this test is failing for some reason. `ar.picks_` is broken ... it's also the case in the non-rebased PR